pymeasure/instruments/keithley/keithley2750.py

```diff
@@ -0,0 +1,95 @@
+#
```

Comment:
All instruments are required to have accompanying documentation. See the [docs for adding documentation](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#adding-documentation), and feel free to based it off of the other Keithley instruments..